packages/styles/scss/components/_index.scss

```diff
@@ -41,6 +41,7 @@
 @use 'radio-button';
 @use 'search';
 @use 'select';
+@use 'skeleton' as skeleton-styles;
```

Comment:
note for reviewers: without `as skeleton-styles` the it was giving me an error about there already being a skeleton file (`data-table/skeleton` line 19). Wasn't sure if this was ok to do.. or should I rename the file to be something else?